Sources/WebSocket/WebSocket.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import Core
+import HTTP
+import Crypto
+import TCP
+
+public class WebSocket {
```

Comment:
We might need to call this `WebSocket.Socket` to avoid name collision between module and class name.